dali_mqtt_daemon.py

```diff
@@ -87,6 +87,31 @@ def dali_scan(driver):
     return lamps
 
 
+def scan_groups(dali_driver, lamps):
+    logger.info("Scanning for groups")
+    groups = dict.fromkeys(range(16), [])
+    for lamp in lamps:
+        try:
+            logging.debug("Search for groups for Lamp {}".format(lamp))
+            group1 = dali_driver.send(gear.QueryGroupsZeroToSeven(address.Short(lamp))).value.as_integer
+            group2 = dali_driver.send(gear.QueryGroupsEightToFifteen(address.Short(lamp))).value.as_integer
+
+            lamp_groups = []
+
+            for i in range(8):
+                if (group1 & 1<<i) != 0:
+                    groups[i].append(lamp)
+                    lamp_groups.append(i)
+            for i in range(8):
```

Comment:
We could probably do everything in a single for

Reply:
Sure, that would make sense. I just took the code over and adopted it slightly so that variables matched. Have no means to test the group functionality at the moment.